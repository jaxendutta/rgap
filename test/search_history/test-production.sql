SELECT * FROM SearchHistory WHERE user_id = 5 ORDER BY search_time DESC LIMIT 1;

DELETE FROM SearchHistory WHERE history_id = 240;

SELECT * FROM SearchHistory WHERE history_id = 240;
