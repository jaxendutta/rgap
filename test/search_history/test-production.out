+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|history_id | user_id |search_recipient | search_grant | search_institution     | search_filters | search_time         | result_count | saved |
+---------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+
|       240 |       5 | professor       | scholarship  |                   NULL |           NULL | 2025-03-14 20:50:49 |          300 | false |
+-----------+---------------------+-------------+--------------------------------------------------------------+-----------------------+-------+

Query OK, 1 row affected (0.10 sec)

Empty set (0.00 sec)
